sdk/core/Azure.Core/api/Azure.Core.net461.cs

```diff
@@ -200,6 +200,20 @@ public partial class RequestContext
         public void AddPolicy(Azure.Core.Pipeline.HttpPipelinePolicy policy, Azure.Core.HttpPipelinePosition position) { }
         public static implicit operator Azure.RequestContext (Azure.ErrorOptions options) { throw null; }
     }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public partial struct RequestFailedDetails
```

Comment:
I'm wondering if we need this as a public type.  It seems like RequestFailedDetailsParser could have two methods: GetResponseError(Response response) and GetAdditionalInfo(), and we could reduce the number of types we're adding to the API.

Reply:
Unless I am misunderstanding the suggestion, this would mean that we'd have to process the response content twice - once to extract the error details and once to extract the additionalInfo collection. I not sure it's worth doing that just to avoid the type.

Reply:
In the latest iteration, I eliminated this type.